Yehor_Dzhurynskyi/5/dungeon_game.py

```diff
@@ -0,0 +1,112 @@
+import math
+import random
+import level
+import dungeon_game_logger
+
+MOVES = {
+    'left': (-1, 0),
+    'right': (1, 0),
+    'up': (0, -1),
+    'down': (0, 1)
+}
+
+
+def offer_moves(lvl, player_x, player_y):
+    """
+    offer player to do move
+        :param str lvl: string representation of level
+        :param int player_x: current x-position of player
+        :param int player_y: current y-position of player
+        :return: delta x, y coords
+        :rtype: tuple
+    """
+
+    dungeon_game_logger.logger.debug('offering moves to player')
+    lvl_size = level.size(lvl)
+
+    up_cell = level.get_cell(lvl, player_x, player_y - 1)
+    down_cell = level.get_cell(lvl, player_x, player_y + 1)
+    left_cell = level.get_cell(lvl, player_x - 1, player_y)
+    right_cell = level.get_cell(lvl, player_x + 1, player_y)
+
+    message = f'# - wall (dead end)\n$ - treasure\n! - trap\n_ - empty space\np - player\n'
+    env_message = '{:^3}\n{}p{}\n{:^3}\n'.format(up_cell, left_cell, right_cell, down_cell)
+
+    dungeon_game_logger.logger.info(message)
+
+    if up_cell == '!' or down_cell == '!' or left_cell == '!' or right_cell == '!':
```

Comment:
Instead of this you could create collection object with all the cells and then just check if '!' in collection.